charts/substra-backend/values.yaml

```diff
@@ -66,7 +66,7 @@ server:
   image:
     registry: ghcr.io
     repository: substra/substra-backend
-    tag: 0.32.0
+    tag: &backend-image-tag 0.33.0
```

Comment:
Nice :p

Reply:
Really nice, didn't think about this use of anchor